src/mixins/DepcomMixin.js

```diff
@@ -1,37 +1,34 @@
 import Commune from "../lib/Commune"
 
 export default {
-  asyncComputed: {
-    communes: {
-      get: function () {
-        if (!this.codePostal || this.codePostal.toString().length !== 5) {
+  methods: {
+    fetchCommune: async function () {
```

Comment:
Il y a un sujet autour de la bonne gestion de la récupération des données de communes.
En cas de saisie rapide, ou de délai de récupération des infos il peut y avoir des incohérences ou des erreurs.

Par exemple, la saisie d'un code postal valide suivi de la bonne récupération suivi de la modification du CP par une valeur invalide et une validation de l'écran dans la foulée. Cela génère une situation où l'on génère une modification du store inadaptée

```json
{
    "entityName": "menage",
    "fieldName": "depcom",
    "value": {
        "_codePostal": "12345",
        "_departement": "55",
        "_epci": "200049187",
        "_epciType": "CA",
        "_nomCommune": "Champneuville",
        "_region": "44",
        "depcom": "55099"
    }
}
```

On peut mettre ça hors de cette PR mais j'ai fait un [ticket](https://trello.com/c/yhE3oeST/468-corriger-la-corruption-potentielle-des-infos-de-commune) sur Trello.

Reply:
C'est noté, merci